src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+
+  const longTerm = 7;
+  const longTermDiscount = 50;
```

Comment:
Great job:)

shortTerm = SHORT_TERM

Just a minor comment. As a tradition, all constant values should be assigned in uppercase with 
underscore (like in the example above)